client/components/Icon/customIcons.tsx

```diff
@@ -417,4 +417,20 @@ export default {
 		),
 		viewboxDefault: 24,
 	},
+	at: {
+		path: (
+			<text
+				fontSize="22px"
+				style={{
+					fontSize: 22,
+					fontFamily:
+						"'BlinkMacSystemFont', 'Segoe UI', 'Roboto', 'Oxygen', 'Ubuntu', 'Cantarell', 'Open Sans', 'Helvetica Neue', sans-serif",
+				}}
+				transform="translate(0, 16)"
+			>
+				@
+			</text>
+		),
+		viewboxDefault: 20,
+	},
```

Comment:
I couldn't figure out a way to just get an `@` rendered in the editor toolbar without using this SVG "hack".

Reply:
<img width="636" alt="image" src="https://user-images.githubusercontent.com/2208769/94860306-e2b72b00-0403-11eb-87d8-afa5ae90cfe7.png">

We may want to straighten it out in its viewbox a little so it looks centered when the toolbar element is active. I would prefer to just create a prebaked SVG of this (and I'm happy to do so).

Thinking aloud some more, I also wonder if `@` is the best piece of iconography for this, and even whether that's the best character to use to trigger this feature, since we will eventually implement at-mentions in discussions (and maybe other Prosemirror content) and I don't know whether that will be an overloading of this feature or not. I would suggest `#` is at least as good since it leaves `@` available for social functionality, and since `#` evokes the hash at the end of a link.

Reply:
I totally agree, @ seems kind of odd since it's conventionally used for social mentions. I'll leave the SVG viewbox alignment out for now until we figure out what icon/trigger symbol we want to use. I'm leaning against `#` for the same reasons you mentioned. How about `&`? I'd argue it could be a good contender since it's not too frequently used and gets a couple bonus points for its similar semantics in some programming languages.